Tools/Scripts/webkitpy/__init__.py

```diff
@@ -34,8 +34,8 @@
 from webkitcorepy import AutoInstall, Package, Version
 AutoInstall.set_directory(os.path.join(libraries, 'autoinstalled', 'python-{}-{}'.format(sys.version_info[0], platform.machine())))
 
-if sys.version_info >= (3, 7):
-    AutoInstall.register(Package('pylint', Version(2, 6, 0)))
+if sys.version_info >= (3, 7, 2):
+    AutoInstall.register(Package('pylint', Version(2, 17, 5)))
```

Comment:
Such a big version bump makes me somewhat nervous, given it makes it more likely we'll end up with different lints being found on different versions of Python…

https://github.com/pylint-dev/pylint/commit/a628d6908c49271302daca1b43091043aa67d8bc looks like it fixes the actual issue, which suggests pylint 2.7.0 should work, at least to avoid the initial failure?

Reply:
This seemed like a good idea, so I tried it, but unfortunately it doesn't work:

```
  File "/home/mcatanzaro/Projects/WebKit/Tools/Scripts/libraries/webkitcorepy/webkitcorepy/autoinstall.py", line 305, in install
    raise ValueError('No archives for {}-{} found'.format(self.pypi_name, self.version))
ValueError: No archives for pylint-2.7 found
```

I have no idea why this is happening. Using trial and error, I discovered that versions 2.7 through 2.12 are all unavailable. So 2.13 seems to be the oldest usable version. I'll try 2.13.9, which supports python 3.6.2 and newer. This seems to work.

Reply:
Apparently I filed a bug for this in 2021: https://bugs.webkit.org/show_bug.cgi?id=223603